gengokit/httptransport/templates.go

```diff
@@ -233,6 +237,7 @@ var (
 	_ = ioutil.NopCloser
 	_ = pb.Register{{.Service.Name}}Server
 	_ = io.Copy
+	Verbose = false
```

Comment:
Could this be in its own declaration block with a comment? 

Perhaps a comment could be added to the `_` block and mention it is for compiling even if those packages are not used.